### PR TITLE
Added http/https toggle

### PR DESCRIPTION
**Pull request by danielcj2**
Design will likely change, just added a placeholder button toggle for now.

### CODE REVIEW

File: Client/src/Pages/Monitors/CreateMonitor/index.jsx [hunk 1]
```diff
@@ -199,15 +200,47 @@ const CreateMonitor = () => {
...
+                    level="secondary"
+                    label="HTTPS"
+                    onClick={() =>
+                      setChecks((prev) => ({ ...prev, type: "https" }))
+                    }
+                    sx={{
+                      backgroundColor:
+                        checks.type === "https" &&
+                        theme.palette.otherColors.fillGray,
+                    }}
```
Comment (ajhollid) [inline Client/src/Pages/Monitors/CreateMonitor/index.jsx:237]: It looks like the `handleCreateMonitor` method hasn't been updated and still just prepends `https` in all cases?  Did I miss something somewhere?

**Review by ajhollid — COMMENTED** [review]
When `ping` type is selected can we hide the `http` prefix in the input box for URL entirely?  Or replace it with `icmp` or `ping` something appropriate? 

It looks a bit misleading atm since it doesn't use `http` or `https` protocol

File: Client/src/Pages/Monitors/CreateMonitor/index.jsx (hunk 1 above)
Comment (danielcj2) [reply Client/src/Pages/Monitors/CreateMonitor/index.jsx:237]: Woops, looks like I forgot to update it.

**Review by ajhollid — APPROVED** [review]
Nice, everything is working!

**Pull request merged**